miniscope_io/models/devupdate.py

```diff
@@ -74,7 +74,9 @@ def validate_values(cls, values: dict) -> dict:
         elif target in UpdateTarget:
             raise NotImplementedError()
         else:
-            raise ValueError(f"{target} is not a valid update target, need an instance of UpdateTarget")
+            raise ValueError(
+                f"{target} is not a valid update target," "need an instance of UpdateTarget"
```

Comment:
this happens to me literally all the time where i want to split a line for aesthetic reasons and then black is like "no you should not have split that line." it borders on malicious compliance and i imagine `black` having like a smile on its face every time it does this like "you were the one who set the line width!"

Reply:
Oh, I was fully trusting `black` and wasn't even paying attention to the changes it made lol.

Reply:
i think it is literally equivalent to the interpreter after parsing (?), but at the same time there is something bad about `("start " "end")` vs `"start end"` that feels like the machines are taking revenge when they make it happen.